app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -10,7 +10,8 @@ class ManageIQ::Providers::Openshift::ContainerManager < ManageIQ::Providers::Co
   has_many :container_routes,                      :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_projects,                    :foreign_key => :ems_id, :dependent => :destroy
 
-  default_value_for :port, 8443
+  DEFAULT_PORT = 8443
+  default_value_for :port, DEFAULT_PORT
```

Comment:
> @abonas isnt it 8663 for k8s and 6443 for openshift?

nope
k8s is 6443
https://github.com/kubernetes/kubernetes/blob/cff5832e7c258066c42e40c8f09bc1ee05d831e0/docs/admin/accessing-the-api.md

ose is 8443
https://github.com/openshift/origin/blob/f59053a0a62249190a721ab6a67d45986d80f596/pkg/cmd/server/start/listen_arg.go